libcodechecker/server/instance_manager.py

```diff
@@ -70,10 +73,14 @@ def __rewriteInstanceFile(append, remove, folder=None):
         #
         # Also, we remove the records to the given PIDs, if any exists.
         append_pids = [i['pid'] for i in append]
-        instances = [i for i in json.load(f)
-                     if i['pid'] not in append_pids and
-                     (i['hostname'] + ":" + str(i['pid'])) not in remove and
-                     __checkInstance(i['hostname'], i['pid'])]
+        try:
```

Comment:
We have a function in our util to handle invalid json files: https://github.com/Ericsson/codechecker/blob/2630549c542f1d6a385ba85f906703e2fb63b368/libcodechecker/util.py#L552
It would be better to use this.

Reply:
@sylvestre Do you plan to update this pull request? If yes please fix pycodestyle error:
```
libcodechecker/server/instance_manager.py:79:80: E501 line too long (80 > 79 characters)
```
**Note**: I have created a separate pull request (#1695) to handle invalid json files properly in all places. So if you fix the style error we will accept this pull request.